manim/utils/space_ops.py

```diff
@@ -561,16 +507,19 @@ def midpoint(
     return center_of_mass([point1, point2])
 
 
-def line_intersection(line1: Sequence[float], line2: Sequence[float]) -> np.ndarray:
-    """Returns intersection point of two lines, each defined with
+def line_intersection(
+    line1: Sequence[np.ndarray], line2: Sequence[np.ndarray]
+) -> np.ndarray:
+    """Returns intersection point of two line segments, each defined with
```

Comment:
```suggestion
    """Returns intersection point of two lines, each defined with
```
The formulas actually take into account the line defined by the vectors, and not the lne segment.
For Example:
![Test_ManimCE_v0 14 0](https://user-images.githubusercontent.com/90276965/152633614-ee8eac33-b188-4826-8b8f-4751d97fb0d7.png)
The above line segments do not intersect but the lines do:
```py
>>> print(line_intersection([[-1,0,0], [0,3,0]],[[1,2,0], [-1,3,0]]))
[-0.14285714  2.57142857  0.        ] 
```

Reply:
resolved by latest commit